test/jdk/jdk/modules/incubator/DefaultImage.java

```diff
@@ -108,10 +109,8 @@ public void testAllSystem() throws Throwable {
     static ToolResult java(String... opts) throws Throwable {
         ByteArrayOutputStream baos = new ByteArrayOutputStream();
         PrintStream ps = new PrintStream(baos);
-        String[] options = Stream.concat(Stream.of(getJava()), Stream.of(opts))
-                .toArray(String[]::new);
 
-        ProcessBuilder pb = new ProcessBuilder(options);
+        ProcessBuilder pb = ProcessTools.createLimitedTestJavaProcessBuilder(opts);
```

Comment:
Shouldn't the test be marked as flagless?

Reply:
Fixed.  thanks